### DISCUSSION TITLE
About the velocity result: the velocity is opposite to the expression result

### DISCUSSION THREAD

**Discussion by Caizhengwen123** [theislab/scvelo#778] [Q&A]
Mostly the gene velocity is same tendency as the expression result like your document showed, but some of my data analyse result show that the some rank_velocity_genes are opposite, and I don't know how to expression this kind of phenomenon, can you help me to figure out what matter of it? thanks a lot.
![image](https://user-images.githubusercontent.com/22273298/146721199-c2e6950e-3d38-440d-8ddc-c29337c2554b.png)

**Comment by WeilerP** [comment]
There have been numerous detailed discussion on this. Please have a look at e.g. #216 or #462.

**Reply by Caizhengwen123** [reply]
I'm sorry to disturb you again. The other two Issue you recommended are not very relevant to what I have mentioned. I mainly want to know why the velocity increases but the expression is not high.

**Reply by WeilerP** [reply]
What I was trying to say: You should not/cannot draw any conclusions from the inferred velocities as the data used is of poor quality (for RNA velocity analysis).
